app/src/androidTest/java/org/simple/clinic/di/TestAppComponent.kt

```diff
@@ -38,12 +38,11 @@ import org.simple.clinic.util.identifierdisplay.IdentifierDisplayAdapterAndroidT
 @AppScope
 @Component(
     modules = [
-      AppModule::class,
-      FakerModule::class,
-      TestClockModule::class
```

Comment:
Curious, why are we removing the `TestClockModule`?

Reply:
NVM.

Reply:
In a later commit, this gets included in the `TestAppModule` instead of being on the component.